modules/login/php/requestaccount.class.inc

```diff
@@ -33,6 +33,10 @@ class RequestAccount extends \NDB_Form
     function setup()
     {
         $this->tpl_data['page_title'] = 'Request Account';
+        $this->tpl_data['captcha_key'] = \NDB_Factory::singleton()
+             ->config()
+             ->getSetting('reCAPTCHAPublic');
+        $this->tpl_data['nonce'] = base64_encode(openssl_random_pseudo_bytes(16));
```

Comment:
This is required as a security measure to avoid making our Content Security Policy weaker. Detailed discussion here. https://github.com/google/recaptcha/issues/107